CHANGELOG.md

```diff
@@ -4,7 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!--## Unreleased-->
+## Unreleased
+
+- **BREAKING** Changed interface to:
```

Comment:
Are any of these actually breaking changes? None of these properties are part of the public API, I would think? So maybe nothing to put in the changelog actually (except maybe that it compiles with more strict flags, though we still don't know which ones).

Reply:
well the global `Element` part would be public breaking

Reply:
For which use case?